pelican/plugins/image_process/image_process.py

```diff
@@ -220,6 +296,10 @@ def harvest_images_in_fragment(fragment, settings):
     parser = settings.get("IMAGE_PROCESS_PARSER", "html.parser")
     soup = BeautifulSoup(fragment, parser)
 
+    copy_exif_tags = settings.get("IMAGE_PROCESS_COPY_EXIF_TAGS", False)
+    if copy_exif_tags:
+        ExifTool.instance = ExifTool()
```

Comment:
Perhaps somewhere around here we should also check to make sure the `exiftool` executable is actually installed? Something like:
```python
import shutil

if shutil.which("exiftool") is None:
    # [print user-friendly warning that EXIF tags were not copied due to missing `exiftools`]
else:
    […]
```

Reply:
fixed